spec/models/defence_request_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe DefenceRequest, type: :model do
+  subject { DefenceRequest.new }
```

Comment:
You can override it where it's needed - don't see anything in this spec that needs it.